modules/platforms/cpp/tests/odbc-test/connection_test.cpp

```diff
@@ -174,3 +174,34 @@ TEST_F(connection_test, auth_connection_incorrect_creds) {
     test_conn_str(get_incorrect_secret_auth_connection_string());
     test_conn_str(get_incorrect_auth_connection_string());
 }
+
+TEST_F(connection_test, dbms_version) {
+    set_authentication_enabled(false);
```

Comment:
Let's move authentication-related tests to separate suite. This does not look right.